module-1/m14-wev-dev-environment/muder-mystery.md

```diff
@@ -4,4 +4,153 @@ author: Tuan Hoang
 description: solutions for full stack web assignments
 ---
 
-[Back to README](../README.md)
+
+# Murder Mystery
+Init: Open file README.md
+Open your terminal
+
+ 
+#### 00 
+1. `cd desktop/cmystery`
+2. `cat step0`
+
+3. `Cmd +N`: Open a new terminal to navigate around and testing new commandlines. 
+
+By the time, there are two opened Terminal, one for reading instructions one for testing commandlines and navigate files.
+
+![](https://i.imgur.com/jPQwZr5.png)
+
+
+Check up folder `Detective Handbook`
+Check up folder `mystery`
+
+
+#### 01
+
+------
+From: clmystery/
+------
+
+1. `cat step1`
+2. `less mystery/crimescene` : to view content in different viewer.
+3. `q` : exit viewer
+
+4. `touch clues.txt`
+
+5. `grep CLUE mystery/crimescene > clues.txt`
+
+**CLUES:**
+*CLUE: Footage from an ATM security camera is blurry but shows that the perpetrator is a tall male, at least 6'.
+CLUE: Found a wallet believed to belong to the killer: no ID, just loose change, and membership cards for AAA, Delta SkyMiles, the local library, and the Museum of Bash History. The cards are totally untraceable and have no name, for some reason.
+CLUE: Questioned the barista at the local coffee shop. He said a woman left right before they heard the shots. The name on her latte was Annabel, she had blond spiky hair and a New Zealand accent.*
+
+6. `git add .``
+7. `git commit -m "Upload clues"``
+8. `git push` 
+
+#### 02
+
+
+1. `cat step2`
+2. `touch possible_witnesses.txt`
+3. `grep Annabel mystery/people > possible_witnesses.js`
+
+4. `git add .``
+5. `git commit -m "Upload  witnessses"`
+6. `git push`
+
+#### 03
+
+**CLUES:**
+*Annabel Sun     F       26      Hart Place, line 40
+Oluwasegun Annabel      M       37      Mattapan Street, line 173
+Annabel Church  F       38      Buckingham Place, line 179*
+
+
+1. `cat step3`
+2. `ls mystery/streets/Buckingham_*`
+3. `less mystery/streets/Buckingham_Place`
+4. `q`
+5. `ls mystery/streets/Hart_*`
+6. `less mystery/streets/Hart_Place`
+7. `q`
+
+#### 04
+
+
+1. `cat step4`
+2. `touch interview.txt`
+3. `head -179 mystery/streets/Buckingham_Place | head -40 mystery/streets/Hart_Place > interview.txt `
```

Comment:
I see you have stopped giving explanation per course. However, for pipelined command like this, learner may need a tiny description.